notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>It is generally advisable to avoid <code>from module import *</code> and instead only import the needed objects or use <code>import module as alias.</code> This avoids potential conflicts and especially makes it easier for a reader to see where the imported objects/functions used in the notebook come from, if they wanted to go look at the source code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thank you! Import statements are correctly declared through the notebook. </p><p> </p><p> </p>

Reply:
<p>closing conversation, resolved</p>